pyaedt/edb.py

```diff
@@ -3668,13 +3657,126 @@ def create_port(self, terminal, ref_terminal=None, is_circuit_port=False):
         -------
 
         """
-        if not ref_terminal:
-            port = CoaxPort(self, terminal._edb_object)
-        else:
-            if is_circuit_port:
-                port = CircuitPort(self, terminal._edb_object)
-            else:
-                port = GapPort(self, terminal._edb_object)
-            port.ref_terminal = ref_terminal
-            port.is_circuit_port = is_circuit_port
-        return port
+
+        terminal.boundary_type = "PortBoundary"
+        terminal.is_circuit_port = is_circuit_port
+
+        if ref_terminal:
+            ref_terminal.boundary_type = "PortBoundary"
+            terminal.ref_terminal = ref_terminal
+
+        return self.ports[terminal.name]
+
+    @pyaedt_function_handler
+    def create_voltage_probe(self, terminal, ref_terminal):
```

Comment:
I think it would be better to move this to edb_data/source.py. I understand it was already here but this bring edb.py a bit messy.